### PR TITLE
 Add new qa test scenario for neutron dvr.

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+          if [ "$ret" = "0" ]; then
+            ssh root@$admin "
+            export cloud=$cloud ;
+            export TESTHEAD=$TESTHEAD ;
+            export cloudsource=$cloudsource ;
+            export nodenumber=$nodenumber ;
+            export hacloud=$hacloud ;
+            export clusterconfig=$clusterconfig ;
+            export want_ceph=1 ;
+            export want_dvr =1;
```
Comment (rhafer) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml]: nitpick there shouldn't be a space before the ```=```. For consistency reasons you might want to add one before the ```;```

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      <b>This job is managed by JJB! Changes must be done in
+      <a href='https://github.com/SUSE-Cloud/automation/tree/master/scripts/jenkins/qa/'>git</a>
+      </b>
+      This job will redeploy scenario-9:
+        - 8 nodes
+        - HA with SBD (2 nodes)
+        - ceph (3 nodes)
+        - 3 kvm compute nodes
```
Comment (rhafer) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-9.yaml]: All the other jobs seem to use 2 compute nodes. Is there a special reason for needing 3 here? Also as far as I can see here this is using the ```hw_number``` 2 are you sure that that setup really has enough nodes? @vmoravec could you clarify on this?

**Comment by rhafer** [comment]
looking at the diff between cloud-mkphyscloud-qa-scenario-9.yaml and the other scenario files it seem you removed a lot of empty lines from the file. I assume those were added for better readability, please add them here as well.

**Comment by gosipyan** [comment]
@rhafer  https://trello.com/c/rnRorToG according to this card if I understood correctly there should be one ha with two nodes ,3 kvm compute and 3 ceph . But maybe i am wrong .And qa2 should have 8 vm's available.

**Comment by rhafer** [comment]
> And qa2 should have 8 vm's available.

vm's? Wasn't this suposed to be a physical setup?

**Comment by rhafer** [comment]
> according to this card if I understood correctly there should be one ha with two nodes ,3 kvm compute and 3 ceph

Ok. But  then you'd  adjust default for "nodenumber", right?

**Comment by gosipyan** [comment]
@rhafer I am sorry i mean physical machines . but @vmoravec will know better.

**Comment by rhafer** [comment]
BTW, I am pretty sure this job will still fail because of: https://github.com/crowbar/crowbar-openstack/blob/master/crowbar_framework/app/models/neutron_service.rb#L255 (but I also think it's not your job to fix that :) )

Did you try running it somewhere already?

**Comment by gosipyan** [comment]
@rhafer unfortunately I don’t have available psychical hardware i am trying run in on our virtual env but as there is not much resources i adopt it to 4 nodes.Not have successful results yet :/

**Comment by gosipyan** [comment]
@vmoravec  test

**Comment by rhafer** [comment]
Given the HA limitation we still have around dvr (https://bugzilla.suse.com/show_bug.cgi?id=974495), I think this one is ok. +1

**Comment by dirkmueller** [comment]
+1

**Comment by gosipyan** [comment]
As I didn’t succeed with having scenario green in CI I will not merge it yet .Thanks for review!

**Pull request merged**